app/domain/authentication/authn_jwt/orchestrate_jwt_authentication.rb

```diff
@@ -6,14 +6,16 @@ module AuthnJwt
 
     OrchestrateAuthentication ||= CommandClass.new(
       dependencies: {
+        uri_based_parameters_validator: Authentication::AuthnJwt::ValidateUriBasedParameters.new,
```

Comment:
please call this `validate_uri_based_parameters` to match the class name

Reply:
Done